tests/client/test_proxies.py

```diff
@@ -1,6 +1,7 @@
 import pytest
 
 import httpx
+from httpx import URL
```

Comment:
`URL` is a public piece of API; in that case we tend to have tests stick to the `httpx.*` usage we recommend for users. :-)

```suggestion
```